api/factories/file_factory.py

```diff
@@ -169,6 +169,7 @@ def _get_remote_file_info(url: str):
     mime_type = mimetypes.guess_type(url)[0] or ""
     file_size = -1
     filename = url.split("/")[-1].split("?")[0] or "unknown_file"
+    mime_type = mime_type or mimetypes.guess_type(filename)[0]
```

Comment:
I think we can use this way directly and remove L169, would you mind making this change?

Reply:
I agree. I will create a new pull request.